docs/development/privacy.md

```diff
@@ -0,0 +1,49 @@
+# Privacy Checklist for ILIAS Development
+
+Start with a quick self-check. Before diving into specific questions, take a moment to reflect on your approach:
+
+- [ ] Have I considered all data protection-relevant aspects of this feature?
+- [ ] Could I clearly explain my solution to a data protection officer?
+- [ ] Would I feel comfortable using this feature on a platform that processes my own data?
+
+If you can confidently answer “yes” to all of the above – great! If not, the checklist below will help identify areas where your implementation can become more privacy-friendly.
+
+### 1. Data Collection
+- [ ] Are personal data being processed? If so, which data exactly?
+- [ ] Is the processing technically necessary – or just “nice to have”?
```

Comment:
Please consider using language that refers to the logic of Art5 that data processing is tied to a purpose and that the GDPR has a risk-based approach. I am not sure if this question is helpful to prompt this kind of reasoning.